src/controllers/user.js

```diff
@@ -4,6 +4,7 @@ import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
 export const create = async (req, res) => {
   const userToCreate = await User.fromJson(req.body)
 
+  console.log("user to create", userToCreate)
```

Comment:
remove console logs once we are done with the issue and are ready to commit to production :)